lib/config/config.go

```diff
@@ -56,7 +57,12 @@ var (
 	// DefaultDiscoveryServers should be substituted when the configuration
 	// contains <globalAnnounceServer>default</globalAnnounceServer>.
 	DefaultDiscoveryServers = append(DefaultDiscoveryServersV4, DefaultDiscoveryServersV6...)
-
+	// DefaultStunServers should be substituted when the configuration
+	// contains <stunServer>default</stunServer>.
+	DefaultStunServers = []string{
+		"stun.ekiga.net:3478",
+		"stun1.voiceeclipse.net:3478",
+	}
```

Comment:
Maybe add a few more from the lists of public ones, and randomize the list before using? I think we are small enough to not have that much of an impact, and these things seem public enough.